### PR TITLE
Set up custom enricher in Docker environment

### LINKED ISSUE #22: Add prinz as a real enricher inside nussknacker environment
Try to add our plugin to nussknacker environment and see it in the UI as some flow block that would allow to receive and send data in the flow of nussknacker data processing workflow. Try to maybe add it manually by using docker bash console and in the next step try to integrate the build process with injection of built plugin to nussknacker environment automagically.

### PR DESCRIPTION
**Pull request by kantoniak**
* Migrate from a local Docker image to `touk/nussknacker:latest`
* Make Nussknacker use custom `ProcessConfigCreator` from `prinz-sample`

Closes #22.

### CODE REVIEW

File: dev-environment/nussknacker/opt/prinz-sample/base-application.conf [hunk 1]
```diff
@@ -0,0 +1,82 @@
+storageDir: ./storage
```
Comment (kantoniak) [inline dev-environment/nussknacker/opt/prinz-sample/base-application.conf:2]: Directory `/nussknacker-dist/src/universal/conf/` which contains configs is not available in the image.

File: dev-environment/create_environment.sh [hunk 2]
```diff
@@ -1,4 +1,11 @@
 #!/bin/sh
 
+# Compile and package prinz if not packaged yet
+cd '..' &&
+sbt prinz/package prinz_sample/package &&
```
Comment (avan1235) [inline dev-environment/create_environment.sh]: use sbtwrapper as already added to our repository

File: dev-environment/nussknacker/opt/prinz-sample/prinz-application.conf [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+metricsSettings: {
+  url: ${base.grafanaUrl}"/dashboard/db/$dashboard?theme=dark&var-processName=$process&var-env="${environment}
+  defaultDashboard: "flink-esp"
+  processingTypeToDashboard: {
+    "streaming": "flink-esp"
+  }
+}
+
+# TODO: lightbend config can't include files on root level - move nussknacker config on nk level and get rid of this below
```
Comment (avan1235) [inline dev-environment/nussknacker/opt/prinz-sample/prinz-application.conf:31]: TODO shouldn't be merged - open an isssue or solve in this commit if this is yours,  remove if unuesed

File: prinz_sample/src/main/scala/pl/touk/nussknacker/prinz/sample/SampleConfigCreator.scala [hunk 4]
```diff
@@ -7,9 +7,9 @@ import pl.touk.nussknacker.prinz.enrichers.IrisSpeciesEnricher
 
 class SampleConfigCreator extends EmptyProcessConfigCreator {
 
-  protected def defaultCategory[T](obj: T): WithCategories[T] = WithCategories(obj, "Default")
+  protected def allCategories[T](obj: T): WithCategories[T] = WithCategories(obj, "FraudDetection", "Recommendations")
 
   override def services(processObjectDependencies: ProcessObjectDependencies): Map[String, WithCategories[Service]] = Map(
-    "irisSpeciesEnricher" -> defaultCategory(new IrisSpeciesEnricher)
+    "irisSpeciesEnricher" -> allCategories(new IrisSpeciesEnricher)
```
Comment (avan1235) [inline prinz_sample/src/main/scala/pl/touk/nussknacker/prinz/sample/SampleConfigCreator.scala]: let's change the name of enricher to prinz in this PR please

File: dev-environment/nussknacker/opt/prinz-sample/base-application.conf [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+attachmentsPath: ${storageDir}"/attachments"
+
+proxyUrl: "http://localhost:8081"
+proxyUrl: ${?PROXY_URL}
+
+grafanaUrl: ${proxyUrl}"/grafana"
+grafanaUrl: ${?GRAFANA_URL}
+
+#TODO: Figure out the defaults. It's tricky part, because influxUrl and grafanaUrl can point to same location, but one link is used
```
Comment (avan1235) [inline dev-environment/nussknacker/opt/prinz-sample/base-application.conf:60]: TODO shouldn't be merged - open an isssue or solve in this commit if this is yours, remove if unuesed

**Review by avan1235 — CHANGES_REQUESTED** [review]
small fixes needed as for me, but looks pretty simple good :+1:

File: prinz_sample/src/main/scala/pl/touk/nussknacker/prinz/sample/SampleConfigCreator.scala (hunk 4 above)
Comment (kantoniak) [reply prinz_sample/src/main/scala/pl/touk/nussknacker/prinz/sample/SampleConfigCreator.scala]: Done.

File: dev-environment/create_environment.sh (hunk 2 above)
Comment (kantoniak) [reply dev-environment/create_environment.sh]: Done.

File: dev-environment/nussknacker/opt/prinz-sample/prinz-application.conf (hunk 3 above)
Comment (kantoniak) [reply dev-environment/nussknacker/opt/prinz-sample/prinz-application.conf:31]: This TODO comes from Nussknacker. Let's keep it here for reference and update configs when they do the fix.

File: dev-environment/nussknacker/opt/prinz-sample/base-application.conf (hunk 5 above)
Comment (kantoniak) [reply dev-environment/nussknacker/opt/prinz-sample/base-application.conf:60]: This TODO comes from Nussknacker. Let's keep it here for reference and update configs when they do the fix.

**Comment by kantoniak** [comment]
I'd like to keep TODOs since they come from Nussknacker. Fixed other comments.

**Review by avan1235 — APPROVED** [review]

**Pull request merged**